packages/plugins/Pins/src/store/index.ts

```diff
@@ -85,27 +105,57 @@ const storeModule: PolarModule<PinsState, PinsState> = {
           { deep: true }
         )
       }
+      if (!movable || movable === 'none') {
+        rootGetters.map.addInteraction(move)
+        move.on(
+          'select',
+          ({ selected }) =>
+            (document.body.style.cursor = selected.length
+              ? 'not-allowed'
+              : 'inherit')
```

Comment:
```suggestion
              : '')
```
This change from 76661944c5d423b00a7783e25d3b702e15bd01c5 has no influence on the behaviour if one wants the pin movable, as this interaction is only added to the map if the pin should not be usable.  

Looking at `main`, the issue was already present beforehand and may have something to do with the `Translate` interaction in `makeMarkerDraggable` not being updated properly. Would an issue be the way for handling this?

Reply:
https://github.com/Dataport/polar/issues/15